### PR TITLE
Add a possibility to use aliases in mapFields and mapMethods

### PR DESCRIPTION
**Pull request by lacedon**

### CODE REVIEW

File: src/helpers.ts [hunk 1]
```diff
@@ -1,17 +1,33 @@
-export function mapFields(store: object, fieldNames: string[]): Object {
-  return fieldNames.reduce(
-    (result, fieldName) => Object.assign(result, {
-      [fieldName]: () => store[fieldName]
-    }),
+export type PropList = string[] | { [propAlias: string]: string }
+
+function mapProps(store: object, propNames: PropList, mapProp: (store: object, propName: string) => Function): object {
+  const isArray = propNames instanceof Array;
```
Comment (nighca) [inline src/helpers.ts]: Maybe `Array.isArray(propNames)` is better.

**Comment by nighca** [comment]
@lacedon Thx for your excellent PR! There's one little problem in the code.

**Comment by lacedon** [comment]
@nighca, fixed that.
Also remove extra semicolons.

**Comment by nighca** [comment]
[v0.0.5](https://github.com/nighca/movue/releases/tag/v0.0.5)

**Pull request merged**